src/CGT.jl

```diff
@@ -66,8 +66,7 @@ function period(c::Currency, df::AbstractDataFrame; details=false)
 	if details
 		show(df; allrows=true, allcols=true, summary=false, eltypes=false, show_row_number=false, vlines=:all, newline_at_end=true, hlines=[:begin, :header, :end], filters_col=((_, i) -> cols[i] != :Late,))
 	end
-	println("Gain/Loss: ", gain)
-	println("Tax: ", max(0c, gain) * Tax)
```

Comment:
It's still useful to print this because the 2 periods have different deadlines to pay the tax, so it's useful to know how much you need to pay by the corresponding deadline.

Reply:
Agreed. I think the allowance should also be get into account here.